.github/workflows/main.yml

```diff
@@ -45,9 +45,9 @@ jobs:
         echo "WASI_SDK_PATH=`pwd`/wasi-sdk-16.0" >> $GITHUB_ENV
       if : matrix.os == 'windows-latest'
 
-    - uses: actions/setup-node@v2
+    - uses: actions/setup-node@v3
       with:
-        node-version: '16'
+        node-version-file: "package.json"
```

Comment:
Let's just make this `lts`, then it should be roughly what we want going forward.